src/lfs/server_receiver.lua

```diff
@@ -17,19 +17,17 @@ local function httpReceiver(sck, payload)
   end
   collectgarbage()
 
-  local request = require("httpd_req")(payload)
-  local response = require("httpd_res")()
+  request = require("httpd_req")(payload)
+  response = require("httpd_res")()
```

Comment:
curious what does removing the `local` declaration actually do?

Reply:
This makes the variables global so they can be accessed in the offline callback.